.github/workflows/pr-go-lint.yml

```diff
@@ -22,30 +16,17 @@ permissions:
   # Optional: allow read access to pull request. Use with `only-new-issues` option.
   pull-requests: read
 jobs:
-  golangci:
-    name: lint
+  lint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 2
 
-      - uses: actions/setup-go@v3
+      - uses: ./.github/actions/setup-deps
         with:
-          go-version: 1.17
```

Comment:
You set up `go-version` as an input but never use it. It would be this task that should set it, yeah?